### ISSUE TITLE
Is there any plugin to show log files with highlight ?

### ISSUE THREAD

**Issue by caclp** [Alexey-T/CudaLister#94]
I want a plugin to show log files with highlight, is there a recommendation?

**Comment by Alexey-T** [comment]
CudaText editor - looks like CudaLister. it has 'log files' lexer which works ok for even huge files.

**Comment by caclp** [comment]
> CudaText editor - looks like CudaLister. it has 'log files' lexer which works ok for even huge files.

Thank you, Alexey. I downloaded "CudaText", and I think you are the author too  :)

Another issue, I tried to install plugin, but got error 
"Cannot download: ...://raw.githubusercontent.com/Alexyey-T/CudaText ............ Caused by ... Failed to establish a new connection: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond"
![Uploading image.png…]()

Maybe this host has been blocked by firewall, Do we have another way to download the plugin "log files" ?

**Comment by Alexey-T** [comment]
'Log files' is not a plugin, it is out of the box. it is 'lite lexer' in the folder 'data/lexliblite'.
